pages/thc/[processRef]/review.tsx

```diff
@@ -87,19 +113,31 @@ const ReviewPage: NextPage = () => {
       heading="Review Tenancy and Household Check"
       pausable
     >
-      {sections.map(({ heading, rows }) => (
-        <React.Fragment key={heading}>
-          <Heading level={HeadingLevels.H2}>{heading}</Heading>
-
-          {rows.length ? (
-            <SummaryList
-              rows={(rows as unknown) as { key: string; value: string }[]}
-            />
-          ) : (
-            <Paragraph>Loading...</Paragraph>
-          )}
-        </React.Fragment>
-      ))}
+      {sections
+        .filter(section => section.rows.length)
+        .map(({ heading, rows }) => (
+          <React.Fragment key={heading}>
+            <Heading level={HeadingLevels.H2}>{heading}</Heading>
+
+            {rows.length ? (
+              <SummaryList
+                rows={(rows as unknown) as { key: string; value: string }[]}
+              />
+            ) : (
+              <Paragraph>Loading...</Paragraph>
+            )}
+          </React.Fragment>
+        ))}
+
+      <TextArea
+        value={otherNotes}
+        onValueChange={(note): void => setOtherNotes(note)}
+        required={false}
+        disabled={false}
+        includeCheckbox={true}
+        label={{ value: "Any other notes to be added?" }}
+        name="other-notes"
+      ></TextArea>
```

Comment:
```suggestion
      />
```

Prefer self-closing where possible.